packages/rum-core/src/domain/configuration.ts

```diff
@@ -127,6 +130,7 @@ export function validateAndBuildRumConfiguration(
         ? initConfiguration.defaultPrivacyLevel
         : DefaultPrivacyLevel.MASK,
       customerDataTelemetrySampleRate: 1,
+      traceContextInjection: initConfiguration.traceContextInjection ?? TraceContextInjection.ALL,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding some tests for this configuration option? (like those we have for other configuration options)